src/runner.py

```diff
@@ -32,6 +32,7 @@
 IS_LOCAL_ENV = os.getenv('IS_LOCAL_ENV', 'false') == 'true'
 TARGET_API_ENDPOINT = os.getenv('TARGET_API_ENDPOINT', 'http://localhost:8000/api/')
 TARGET_TAPS_ENDPOINT = os.getenv('TARGET_TAPS_ENDPOINT', f'{TARGET_API_ENDPOINT}taps/')
+XOS_TAPS_ENDPOINT = os.getenv('XOS_TAPS_ENDPOINT', 'https://xos.acmi.net.au/api/taps/')
```

Comment:
Should we set this to a dev friendly default, and then set this var app wide prior to release?

```suggestion
XOS_TAPS_ENDPOINT = os.getenv('XOS_TAPS_ENDPOINT', 'http://localhost:8000/api/taps/')
```

Reply:
@sam-maher-acmi I thought about that, but figured it'd be better to not have to set anything new in production and have one more variable to set in dev (when we're already setting the `TARGET_TAPS_ENDPOINT` anyway.

This will stop us needing to programatically set a new env var for all devices in XOS.

What do you think?

Reply:
@sighmon Yep fine by me in this case.

If setting app wide env vars in XOS is a reoccurring issue maybe we could consider UI or a command to do this for us?

Reply:
@sam-maher-acmi Looking at XOS, we do have Balena application variables after all... so I'm wrong about that sorry.

But we would still have to add the new variable to each production Lens Reader app, which is still a bit more painful than one extra variable in dev.